[docs] using a more inclusive language option (#16603)
expo/expo: docs/pages/guides/permissions.md

When adding Expo and other React Native modules to your project, certain Android permissions might be implied automatically. The modules should only add relevant permissions **required** to use the module, however, sometimes you may want to remove some of these permissions.

> Since the `android.permissions` manifest property doesn't work in the bare workflow- when you want to exclude specific permissions from the build, you have to exclude them in your **AndroidManifest.xml**. You can do that with the `tools:node="remove"` attribute on the `<use-permission>` tag.

Reviewer: The wording here is a little awkward due to the repetition: "when you want to exclude permissions, you have to exclude them".
```suggestion
Since the `android.permissions` manifest property doesn't work in the bare workflow, you'll need to edit **AndroidManifest.xml** to exclude specific permissions from the build. You can do that with the `tools:node="remove"` attribute on the `<use-permission>` tag.
```
